Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -3808,7 +3808,7 @@ final class WorkspaceTests: XCTestCase {
                     products: [
                         MockProduct(name: "Foo", targets: ["Foo"]),
                     ],
-                    versions: ["1.0.0"],
+                    versions: ["1.0.0", "1.1.0"],
```

Comment:
Is this change related?

Reply:
yea it demonstrated the bug unintentionally